spec/trace_location_spec.rb

```diff
@@ -1,7 +1,84 @@
 # frozen_string_literal: true
 
+require 'fileutils'
```

Comment:
I thought it seems this statement is better to move to spec_helper.rb

Reply:
Thank you for this suggestion!(And sorry for the delay in replaying)
I'll move this statement to spec_helper in next PR.